e2e/binary_test.go

```diff
@@ -277,21 +277,41 @@ func TestPackBinary(t *testing.T) {
 
 func TestHelmBinary(t *testing.T) {
 	dockerApp, _ := getBinary(t)
-	assertCommand(t, dockerApp, "helm", "helm", "-s", "myapp.nginx_version=2")
-	chart, _ := ioutil.ReadFile("helm.chart/Chart.yaml")
-	values, _ := ioutil.ReadFile("helm.chart/values.yaml")
-	stack, _ := ioutil.ReadFile("helm.chart/templates/stack.yaml")
+	abs, err := filepath.Abs(".")
+	assert.NilError(t, err)
+	dir := fs.NewDir(t, t.Name(), fs.FromDir(abs))
+	defer dir.Remove()
+
+	result := icmd.RunCmd(icmd.Cmd{
+		Command: []string{dockerApp, "helm", "helm", "-s", "myapp.nginx_version=2"},
+		Dir:     dir.Path(),
+	})
+	result.Assert(t, icmd.Success)
+
+	chart, _ := ioutil.ReadFile(dir.Join("helm.chart/Chart.yaml"))
+	values, _ := ioutil.ReadFile(dir.Join("helm.chart/values.yaml"))
+	stack, _ := ioutil.ReadFile(dir.Join("helm.chart/templates/stack.yaml"))
 	golden.Assert(t, string(chart), "helm-expected.chart/Chart.yaml")
 	golden.Assert(t, string(values), "helm-expected.chart/values.yaml")
 	golden.Assert(t, string(stack), "helm-expected.chart/templates/stack.yaml")
 }
 
 func TestHelmV1Beta1Binary(t *testing.T) {
 	dockerApp, _ := getBinary(t)
-	assertCommand(t, dockerApp, "helm", "helm", "-s", "myapp.nginx_version=2", "--stack-version", "v1beta1")
-	chart, _ := ioutil.ReadFile("helm.chart/Chart.yaml")
-	values, _ := ioutil.ReadFile("helm.chart/values.yaml")
-	stack, _ := ioutil.ReadFile("helm.chart/templates/stack.yaml")
+	abs, err := filepath.Abs(".")
+	assert.NilError(t, err)
+	dir := fs.NewDir(t, t.Name(), fs.FromDir(abs))
+	defer dir.Remove()
```

Comment:
This is duplicated, we could make a small helper for that :angel:

Reply:
PTAL